lib/srv/desktop/tdp/proto.go

```diff
@@ -640,7 +636,8 @@ func DecodeMFA(in byteReader) (*MFA, error) {
 	}
 
 	if length > maxMFADataLength {
-		return nil, trace.BadParameter("mfa challenge data exceeds maximum length")
+		io.CopyN(io.Discard, in, int64(length))
```

Comment:
Ignored intentionally:
```suggestion
		_, _ = io.CopyN(io.Discard, in, int64(length))
```